installers/msi-language/StateDeploy/CustomAction.cs

```diff
@@ -1,15 +1,16 @@
 using Microsoft.Deployment.WindowsInstaller;
 using System;
 using System.Text;
-using System.Diagnostics;
-using System.Threading;
 using System.IO;
 using System.Net;
 using System.Collections.ObjectModel;
 using System.Windows.Forms;
 using System.Linq;
 using System.Web.Script.Serialization;
 using System.Collections.Generic;
+using Newtonsoft.Json;
```

Comment:
Shouldn't this be showing up as binaries? This is a third party dep right?

Reply:
Yes it's a 3rd party package. I _think_ the `NuGet` package manager handles getting the necessary files to build this dependency. I've tested the MSI from the build artifacts and it worked fine. I'll make sure to double check.